src/amo/containers/Home.js

```diff
@@ -1,11 +1,45 @@
-import React from 'react';
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+
+import 'amo/css/Home.scss';
+
+export class HomePageBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object.isRequired,
+  }
 
-export default class Home extends React.Component {
   render() {
+    const { i18n } = this.props;
     return (
-      <div>
-        <h1>AMO Home Page Hello World</h1>
+      <div className="HomePage">
+        <h2 className="HomePage-h2">{i18n.gettext('What do you want Firefox to do?')}</h2>
+        <ul className="HomePage-cat-list">
+          <li className="HomePage-block-ads"><a href="#block-ads"><span>{i18n.gettext('Block Ads')}</span></a></li>
```

Comment:
I'm not convinced moving to a stateless component buys us that much quite yet. Leaving it static may help if we need to move things around and make changes. 

As soon as we decide base this on a data structure we can iterate over then that would make more sense in terms of DRY.